astropy/coordinates/tests/test_angles.py

```diff
@@ -46,11 +45,9 @@ def test_create_angles():
 
     a10 = Angle(3.60827466667, unit=u.hour)
     a11 = Angle("3:36:29.7888000120", unit=u.hour)
-    with pytest.warns(AstropyDeprecationWarning, match="hms_to_hour"):
-        a12 = Angle((3, 36, 29.7888000120), unit=u.hour)  # *must* be a tuple
-    with pytest.warns(AstropyDeprecationWarning, match="hms_to_hour"):
-        # Regression test for #5001
-        a13 = Angle((3, 36, 29.7888000120), unit="hour")
+    a12 = Angle((3, 36, 29.7888000120), unit=u.hour)  # *must* be a tuple
```

Comment:
These two should be removed - we deprecated entering angles like this! I think this is why the tests are failing...

Reply:
The problem is that after removing deprecation code, this usage does not fail. No, it succeeded but with different behavior. I think that should not happen. But how/where do I add in the code to throw exception for such a usage? I think this should now raise `NotImplementedError`.

Reply:
Okay, I kept the logic route but now throws an exception instead, so I think this is resolved.